doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -291,21 +291,22 @@ to a distributed constant load of 10 KN/m from the starting point till
          2               1            2             1               1              1
     5⋅<x>  - 26.0⋅<x - 1>  - 5⋅<x - 2>  + 20⋅<x - 5>  - 22.0⋅<x - 7>  + 8⋅<x - 7.5> 
 
+    >>> b.bc_deflection = [(1, 0), (7, 0)]
     >>> b.slope()
-         3                            3                                           
-    5⋅<x>                2   5⋅<x - 2>              2               2              2
-    ────── - 13.0⋅<x - 1>  - ────────── + 10⋅<x - 5>  - 11.0⋅<x - 7>  + 4⋅<x - 7.5>
-      3                          3                                                
-    ────────────────────────────────────────────────────────────────────────────────
-                                          E⋅I                                     
+         3                            3                                                                
+    5⋅<x>                2   5⋅<x - 2>              2               2              2                   
+    ────── - 13.0⋅<x - 1>  - ────────── + 10⋅<x - 5>  - 11.0⋅<x - 7>  + 4⋅<x - 7.5>  + 28.2916666666667
+      3                          3                                                                     
+    ───────────────────────────────────────────────────────────────────────────────────────────────────
+                                                    E⋅I                                      
```

Comment:
Why did these change, and why didn't these fail in the doctests before?

Reply:
It's not a bug, I just forgot to add `b.bc_deflection = [(1, 0), (7, 0)]`. So it was a different beam problem than in figure before.